plugins/data-sources/abstract-sql-query-service/AbstractQueryService.ts

```diff
@@ -777,8 +777,15 @@ abstract class AbstractQueryService implements ISQLQueryService {
       })
     );
 
+    // Filter out fields that are unsupported.
+    const filteredColumns = this.filterOutColumns(columns);
+
     // @todo: fetch foreign keys before responding
-    return columns as [];
+    return filteredColumns as [];
+  }
+
+  filterOutColumns(columns: Column<SqlColumnOptions>[]) {
```

Comment:
We need to pay a bit more attention to how we name things. `filterOutColumns` is very generic. Also, let's use `public`/`private` because the language supports it.

```suggestion
  public/private filterOutUnsupportedColumns(columns: Column<SqlColumnOptions>[]) {
```